syncd/syncd_flex_counter.cpp

```diff
@@ -982,8 +1015,9 @@ void FlexCounter::collectCounters(
 
         countersTable.set(priorityGroupVidStr, values, "");
     }
+
     // Collect stats for every registered router interface
-    for (const auto &kv: rifCounterIdsMap)
+    for (const auto &kv: m_rifCounterIdsMap)
```

Comment:
what is this fix? should we separate into one pr?

Reply:
No, it is in the scope of pr only. Because locally I changed and validated aginast the 1130 branch, which does not include the rif counters. When cherry-picking to master, I need to take additional care exclusively for rif counters.

https://github.com/Azure/sonic-sairedis/pull/462#discussion_r287582193